test/gitea_test.go

```diff
@@ -55,7 +56,7 @@ func TestGiteaPullRequestPrivateRepository(t *testing.T) {
 			".tekton/pipeline.yaml": "testdata/pipelinerun_git_clone_private-gitea.yaml",
 		},
 	}
-	tgitea.TestPR(t, topts)()
+	defer tgitea.TestPR(t, topts)()
```

Comment:
whats the reason for using `defer` here?
anyhow that was the last line in func execution 🤔

Reply:
this is a cleanup function, if i cleanup in TestPR I would not be able to do things because it would cleanup ns/repo etc.. but yeah since it's the last one it's just not useful, but just add it to be the same for the others to  to avoid bugs and head scraping if i need to add more stuff to that function and forgot there was no defer ;)